modules/ipi-install-troubleshooting-bootstrap-vm-inspecting-logs.adoc

```diff
@@ -61,9 +61,9 @@ If the bootstrap VM cannot access the URL to the images, use the `curl` command
 [core@localhost ~]$ sudo podman ps
 ----
 
-. If there are issues with the pods, check the logs of the containers with issues. To check the log of the `ironic-api`, execute the following:
+. If there are issues with the pods, check the logs of the containers with issues. To check the logs of the `ironic` service, run the following command:
 +
 [source,terminal]
 ----
-[core@localhost ~]$ sudo podman logs <ironic-api>
+[core@localhost ~]$ sudo podman logs <ironic-service>
```

Comment:
Cannot we just use `ironic` here rather than this placeholder?

Reply:
I was not certain if just stating`ironic`fetches the relevant logs, but if it does, I will update it accordingly. Thank you!